lib/safira/contest/contest.ex

```diff
@@ -111,23 +115,54 @@ defmodule Safira.Contest do
   end
 
   def list_redeems_stats do
-    Repo.all(from r in Redeem,
-      join: b in assoc(r, :badge),
-      join: a in assoc(r, :attendee),
-      where: b.type == ^1 and not(a.volunteer) and not(is_nil(a.nickname)),
-      preload: [badge: b, attendee: a])
+    Repo.all(
+      from r in Redeem,
+        join: b in assoc(r, :badge),
+        join: a in assoc(r, :attendee),
+        where: b.type == ^1 and not a.volunteer and not is_nil(a.nickname),
+        preload: [badge: b, attendee: a]
+    )
   end
 
   def get_redeem!(id), do: Repo.get!(Redeem, id)
 
   def get_keys_redeem(attendee_id, badge_id) do
-    Repo.get_by(Redeem, [attendee_id: attendee_id, badge_id: badge_id])
+    Repo.get_by(Redeem, attendee_id: attendee_id, badge_id: badge_id)
```

Comment:
I know this was the formatter, but I don't know if in this case the behaviour is the same. Just check the [docs](https://hexdocs.pm/ecto/Ecto.Repo.html#c:get_by/3).

Reply:
I just got the older version and updated the create_redeem function: which one is responsible for update the token_balance atm.